assets/src/edit-story/components/helpCenter/menu/transitioner.js

```diff
@@ -17,47 +17,59 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
-import { Transition } from 'react-transition-group';
 import styled, { css } from 'styled-components';
 /**
  * Internal dependencies
  */
 import { BEZIER } from '../../../../animation';
+import { ScheduledTransition } from '../scheduledTransition';
 
-const DURATION = 500;
+const DURATION = 800;
+
+const enterSyles = css`
+  opacity: 1;
+  transform: none;
+`;
+const exitSyles = css`
+  position: absolute;
+  bottom: 0;
+  opacity: 0.6;
+  transform: scale(0.96);
+`;
+
+const transitionStyles = {
+  entering: enterSyles,
+  entered: enterSyles,
+  exiting: exitSyles,
+  exited: exitSyles,
+};
 
 const Manager = styled.div`
   position: relative;
   color: ${({ theme }) => theme.colors.fg.primary};
-  transition: ${DURATION / 1000}s transform ${BEZIER.default},
-    ${DURATION / 1000}s opacity ${BEZIER.default};
-  opacity: 0.8;
-  transform: translateX(-40px) scale(0.99);
-
-  ${({ state }) =>
-    ['entered'].includes(state) &&
-    css`
-      opacity: 1;
-      transform: none;
-    `};
+  opacity: 0.6;
+  transform-origin: 50% 50%;
+  transform: scale(0.96);
+  transition: transform ${DURATION}ms ${BEZIER.default},
+    opacity ${DURATION}ms ${BEZIER.default};
+  z-index: 1;
 
-  ${({ state }) =>
-    ['exiting', 'exited'].includes(state) &&
-    css`
-      position: absolute;
-      bottom: 0;
-    `};
+  ${({ state }) => transitionStyles[state]};
 `;
 
 export function Transitioner({ children, ...props }) {
   return (
-    <Transition {...props} timeout={DURATION} mountOnEnter unmountOnExit>
+    <ScheduledTransition
+      {...props}
+      timeout={DURATION}
+      mountOnEnter={true}
+      unmountOnExit={true}
```

Comment:
We can shorten these since they're always `true`
```suggestion
      mountOnEnter
      unmountOnExit
```